cmake/onnxruntime_unittests.cmake

```diff
@@ -169,7 +169,25 @@ function(AddTest)
       MACOSX_BUNDLE_SHORT_VERSION_STRING ${ORT_VERSION}
       XCODE_ATTRIBUTE_ENABLE_BITCODE "NO")
 
-    xctest_add_test(xctest.${_UT_TARGET} ${_UT_TARGET}_xc)
+    # This is a workaround for an Xcode 16 / CMake issue:
```

Comment:
This is currently in an `if (IOS)` branch but could it also apply to macOS if building with xcode?

Reply:
it does apply to macOS too. I repro'd it in a macOS build in a test project. however, for ORT, we only do this XCTest business for iOS builds. macOS builds just run onnxruntime_test_all directly.